_breadcrumbs.scss

```diff
@@ -0,0 +1,116 @@
+// Breadcrumbs
+// ===
+
+
+// Configurable Variables
+// ---
+
+$breadcrumbs__divider-content: '>' !default; // TO DO: include stencil icon
```

Comment:
We totally need to figure out how the Icon component works and how to incorporate it into other components. One (almost) sure thing is Icon will have its own template (for SVG support), which means controlling the content will likely need to be done via the template rather than through CSS.

However, this is a totally reasonable implementation for now.